### PR TITLE
Adding SaaS vs Self-Hosted comparison

### PR DESCRIPTION
**Pull request by omnomagonz**

### CODE REVIEW

File: src/content/saas-vs-selfhosted.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Additionally, you can use the Okteto CLI without using the Okteto Platform, but there are important considerations with that use case.
+
+Some examples of what the Okteto Platform manages are: [configuring secrets](/cloud/secrets/), adding [external resources](/cloud/external-resources) into your development environment, and push container images to the [Okteto Registry](/cloud/registry/). For a more comprehensive list of features, check out our [docs](/self-hosted/).
+
+# Important things to know
+
+| SaaS | Self-Hosted |
+| ---- | ----------- |
+| Fully managed by Okteto and great for individuals or small teams with simpler applications and infrastructure requirements. | Great for larger teams with bigger with more complex applications and a large number of microservices. |
+| Up to XX developers per SaaS instance. | Unlimited number of developers in your Okteto instance. |
```
Comment (omnomagonz) [inline src/content/saas-vs-selfhosted.mdx:23]: @pchico83 Do we have a "recommended" number of users per SaaS instance? I imagine we eventually run into scaling issues and I'm wondering if we could call that out here.

If not, I'll remove this so we can merge the PR

File: src/content/saas-vs-selfhosted.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: SaaS vs. Self-Hosted
+description: The difference between Okteto's product offerings in SaaS and Self-Hosted
+sidebar_label: SaaS vs. Self-Hosted
+id: saas-vs-self-hosted
+---
+
+Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the Okteto Platform. We provide these products in two forms: SaaS (fully managed by Okteto) and Self-Hosted (installed using our Helm chart and fully managed by you). The SaaS and Self-Hosted offerings exist primarily to deliver the Okteto Platform experience and features.
```
Comment (pchico83) [inline src/content/saas-vs-selfhosted.mdx:8]:
```suggestion
Okteto consists of two products: our [CLI](/cloud/okteto-cli) and the Okteto Platform. We provide the Okteto Platform  in two forms: SaaS (fully managed by Okteto) and Self-Hosted (installed using our Helm chart and fully managed by you). The SaaS and Self-Hosted offerings exist primarily to deliver the Okteto Platform experience and features.
```

File: src/content/saas-vs-selfhosted.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# The Okteto Platform
+
+In addition to Okteto's [CLI](/cloud/okteto-cli) we optionally provide the Okteto Platform. This is the other half of our product experience that provides the fully managed infrastructure for your development environments. It's what interprets the [Okteto Manifest](/reference/manifest) to spin up a development environment and automates your workflows. One of the biggest advantages of the Okteto Platform is that we scale the underlying cluster infrastructure so you never have to worry about your development environments having insufficient resources to run even your most complex applications.
+
+Additionally, you can use the Okteto CLI without using the Okteto Platform, but there are important considerations with that use case.
+
+Some examples of what the Okteto Platform manages are: [configuring secrets](/cloud/secrets/), adding [external resources](/cloud/external-resources) into your development environment, and push container images to the [Okteto Registry](/cloud/registry/). For a more comprehensive list of features, check out our [docs](/self-hosted/).
```
Comment (pchico83) [inline src/content/saas-vs-selfhosted.mdx:16]:
```suggestion
Some examples of what the Okteto Platform manages are: [configuring secrets](/cloud/secrets/), adding [external resources](/cloud/external-resources) into your development environment, and build and push container images to the [Okteto Registry](/cloud/registry/). For a more comprehensive list of features, check out our [docs](/self-hosted/).
```

**Review by pchico83 — APPROVED** [review]
Minor comments, looks great!

File: src/content/saas-vs-selfhosted.mdx (hunk 1 above)
Comment (omnomagonz) [reply src/content/saas-vs-selfhosted.mdx:23]:
```suggestion
```

Removing it then for now and we can iterate on it 👍

**Pull request merged**